privacyidea/api/lib/utils.py

```diff
@@ -148,6 +148,12 @@ def send_error(errstring, rid=1, context=None, error_code=-311, details=None):
     return ret
 
 
+def send_file(output, filename):
+    content_type = "application/force-download"
```

Comment:
I'm not sure about the ``application/force-download`` mimetype here. I didn't know it, so I googled it, and [it seems like](https://stackoverflow.com/a/10616753) only shouldn't really use it, but instead set the actual content mimetype. I'm wondering -- should we do that instead? What do you think?

Reply:
Fixed in ba5ab0b